README.md

```diff
@@ -125,19 +156,17 @@ let results = model.rerank("what is panda?", documents, true, None);
 println!("Rerank result: {:?}", results);
 ```
 
-Alternatively, raw `.onnx` files can be loaded through the `UserDefinedEmbeddingModel` struct (for "bring your own" text embedding models) using `TextEmbedding::try_new_from_user_defined(...)`. Similarly, "bring your own" reranking models can be loaded using the `UserDefinedRerankingModel` struct and `TextRerank::try_new_from_user_defined(...)`. For example:
+Alternatively, raw `.onnx` files can be loaded through the `UserDefinedEmbeddingModel` struct (for "bring your own" text embedding models) using `TextEmbedding::try_new_from_user_defined(...)`. Similarly,
+"bring your own" image embedding models can be loaded useing the `UserDefinedImageEmbeddingModel` struct and `ImageEmbedding::try_new_from_user_defined(...)`,
+"bring your own" reranking models can be loaded using the `UserDefinedRerankingModel` struct and `TextRerank::try_new_from_user_defined(...)`.
+For example:
 
 ```rust
 macro_rules! local_model {
```

Comment:
Maybe remove the macro? Since it's simple now.

Reply:
The ·local_model· macro only provides a way to write it and is not included in the crate. I think it can be retained.

Reply:
Right. I meant to simplify the README.

Reply:
cc @lywa1998

Reply:
[`95ef48e` (#101)](https://github.com/Anush008/fastembed-rs/pull/101/commits/95ef48ec5f40e27fc7744ad642e19da68a186542)